datasets/bor_inspecties/dataset.json

```diff
@@ -0,0 +1,302 @@
+{
+  "id": "bor_inspecties",
```

Comment:
Underscore zijn niet mogelijk in de dataset id. Maar, camelCase mag wel, dus er is evt. `borInspecties` van te maken.

Reply:
wordt aangepast